Casandras Portfolio

HolaCarmensita opened the pull request:
Hi! I havnt had time to extract the svgs to files and havn't had the time to clean up sections and components, i totally agree! 

https://holacarmensita.com/hola

HolaCarmensita committed:
adding the basic content and components to the project

HolaCarmensita committed:
added favicon

HolaCarmensita committed:
nu använder Techniques, ProjectCard och projects projects.json istället för hårdkodat/inget innehåll

HolaCarmensita committed:
GetInTouch hämtar data från profile.json

HolaCarmensita committed:
added aria-labels to links and buttons

HolaCarmensita committed:
added tailwind to the project

HolaCarmensita committed:
Almost forgot the footer, I have a simple one now atleast

HolaCarmensita committed:
Okey so the command npx @tailwindcss/cli init -p, did  not work. ChatGPT give me a work around and it seems to work?!

HolaCarmensita committed:
Got my font to work with tailwind...

HolaCarmensita committed:
Editor setup tailwind intelliense

HolaCarmensita committed:
deliteed Tailwind and added Styled Components. Did som cleaning up in the css

HolaCarmensita committed:
tailwind-styled toogle button is now using Styled Components and is also a SwitchButton

HolaCarmensita committed:
basic css in introsection

HolaCarmensita committed:
nav hidden on mobile

HolaCarmensita committed:
Smooth scroll

HolaCarmensita committed:
Forgot to add skills to skillsCategory

HolaCarmensita committed:
made the wacy line under links in Button.jsx

HolaCarmensita committed:
some changes to button.jsx to  look more like figma

HolaCarmensita committed:
Sociala medier ikonerna är färgade och About me ser helt ok ut på mobil...

HolaCarmensita committed:
added a circle in the background in about me section, just trying it out

HolaCarmensita committed:
made a MiddleSection component to fill out the gap between sections as the figma design

HolaCarmensita committed:
styling mobile first for skills done 90%

HolaCarmensita committed:
SkillsCategories ser ut som i figma nu i mobilvy

HolaCarmensita committed:
Projects section looks like the mobile figma design

HolaCarmensita committed:
added animations and trying them out on AboutMe section

HolaCarmensita committed:
IntersectionObserver added to the navigation

HolaCarmensita committed:
added wrapper to the sections, need some more work in te getintouch

HolaCarmensita committed:
Dont remember what i have done, forgot to commit.... worked with buttons and layout

HolaCarmensita committed:
SocialMedia.jsx done responsive and animated

HolaCarmensita committed:
chnaged CSS and layout in about, profileCard and LayoutWrapper

HolaCarmensita committed:
Behövde sära på skills och about me för att få Sociala medierna att hamna på rätt plats... målet är att lägga ihop about och skills till en och samma komponent i App för att få med Resumé-knappen hela vägen

HolaCarmensita committed:
min/max-with fix in the LayoutWrapper.jsx fixed

HolaCarmensita committed:
added some css to Featured Projects, not get fully responsive

HolaCarmensita committed:
Prjects section are now responsive

HolaCarmensita committed:
Svg in skills css added

HolaCarmensita committed:
error when deplying to Netflify, navigation component

HolaCarmensita committed:
CSS for the footer and media query in Skills

HolaCarmensita committed:
animation to HeroAnimation SVG

HolaCarmensita committed:
changes some css in intro

HolaCarmensita committed:
Changed the heroanimation a bit

HolaCarmensita committed:
fixed scroll bugg in the navigation due to added id-tag in the wrong elements and oslo made the Resume-button free from the flex and moved sthe skills section into the about component

HolaCarmensita committed:
delited skilsl from app.jsx

HolaCarmensita committed:
added a animation to the intro

HolaCarmensita committed:
HolaAnimation is done but now I need to refactor to make Heroanimation and HeloAnimation to talk, using state

HolaCarmensita committed:
got an build error due to heroAnimation instead of Heroanimation on import

HolaCarmensita committed:
Jjust some StyledComponents fixes in the svgs and Styled props

HolaCarmensita committed:
Added a UseState to make the Hero know shen to start

HolaCarmensita committed:
added circle in AboutMe as FigmaDesign

HolaCarmensita committed:
changed the color of the rect in heroanimation, hope it will change on build

HolaCarmensita committed:
changed my mind, I want to use svg code instead of img

HolaCarmensita committed:
made the circle a component

HolaCarmensita committed:
fixes bug whee  the sticky was read in as a reactProp in the DOM instead of a StyledProp

HolaCarmensita committed:
added some circles and made a CircleRight component

HolaCarmensita committed:
changed the nav a bit to serach for ids and not sections fot the actoive function. still a bug on parts not having a full vh

HolaCarmensita committed:
Forgot the overflow hidden on the skills wrapper

HolaCarmensita committed:
changed rootMargin to:-50% 0px -50% 0px to active the links scroller from the bottom to

HolaCarmensita committed:
changed the animations[C

HolaCarmensita committed:
Changed some animations

HolaCarmensita committed:
Changing the profileSection

HolaCarmensita committed:
I've lost it, tryin gout alot

HolaCarmensita committed:
Some editing

HolaCarmensita committed:
Just css

HolaCarmensita committed:
Not responsive, have to do alot of groundwork with grid etc but have to wait until i have more time

HolaCarmensita committed:
alot of css

HolaCarmensita committed:
most of the responsivness work here

HolaCarmensita committed:
Just a clamp fix

HolaCarmensita committed:
...

HolaCarmensita committed:
netlift.toml

HolaCarmensita committed:
Fixed bug with the painted wave, painting to early making it to short

HolaCarmensita committed:
Did some flex-work on the projectCard to fit the real projedtimgs

HolaCarmensita committed:
changed imgs

HolaCarmensita committed:
added comments

HolaCarmensita committed:
changed the id from intro to helo, maybe not the best for accessibility. figuring it aout

HolaCarmensita committed:
adding thumbnail etc

HolaCarmensita committed:
change some content text

HolaCarmensita committed:
Changed some text in profile.jsn and gap in ProfileCard

HolaCarmensita committed:
css changes amking the aboutme part full hvh on larger screens.

HolaCarmensita committed:
t.current += 0.2 changed to o.1

HolaCarmensita committed:
I have to take out skillssectionfrom about when i have time to make the layoutwrapper do its real job, now it's different paddings everywere and the layout is not  consistant

HolaCarmensita committed:
Well here we go again, started with one fix and then a saw something else etc, should commit more but it gets to far very fast

HolaCarmensita committed:
made the circles and socialmedia to use the same visibility code, should lay in in utils but not today

HolaCarmensita committed:
knapp på hero arrow

HolaCarmensita committed:
Fixed some props errors and animation on profileCard

HolaCarmensita committed:
Update README.md

HolaCarmensita committed:
Update README.md

HolaCarmensita committed:
added a project

HolaCarmensita committed:
Update projects.json

added project

HolaCarmensita committed:
Add files via upload

HolaCarmensita committed:
Add files via upload

HolaCarmensita committed:
Update projects.json

HolaCarmensita committed:
Update projects.json

HolaCarmensita committed:
changed the layoutwrapper and had to fix the css of every section

HolaCarmensita committed:
Merge branch 'main' of https://github.com/HolaCarmensita/js-project-portfolio

HolaCarmensita committed:
some css

HolaCarmensita committed:
fontsize 16px

HolaCarmensita committed:
gap in projects

HolaCarmensita committed:
adjudted the padding on h2 tags

HolaCarmensita committed:
max-width on desktop smaller

HolaCarmensita committed:
added new resume

HolaCarmensita committed:
forgot the pointer-events on SocialWrapper

HolaCarmensita committed:
the pointer noen moved to the social media not the wrapper

HolaCarmensita committed:
new data

HolaCarmensita committed:
did some changes to the heroAnimation and made it work for phone on landscape

HolaCarmensita committed:
wanted to make the Arrow to work in a flex but need more time, so now it will just show on some devices and work with padding bottom

HolaCarmensita committed:
added a   min-height 100dvh to layoutwrapper, maybe a misstake

HolaCarmensita committed:
added even more ugly padding on arrow to not be halfhidden on phone

HolaCarmensita committed:
added even more ugly padding on arrow to not be halfhidden on phone

HolaCarmensita committed:
added even more ugly padding on arrow to not be halfhidden on phone

HolaCarmensita committed:
added even more ugly padding on arrow to not be halfhidden on phone

HIPPIEKICK commented:
I still don't have access to your Figma file so I can't review

HolaCarmensita commented:
> I still don't have access to your Figma file so I can't review

But i have given you access... so weird. Ill try again

HIPPIEKICK reviewed (changes requested):
You've met all requirements except for the Lighthouse one, see image below. Sort this out and you're good to go. PS. Nice design ⭐ 

<img width="1512" alt="Screenshot 2025-06-09 at 12 05 18" src="https://github.com/user-attachments/assets/aaa0d4bb-9afb-4f7e-9da6-688077963953" />

HolaCarmensita committed:
added a role=img to the div that works as a img due to LightHouse warning

HolaCarmensita committed:
Changed h4 to h3 to have the right order of headings due to LightHouse accessibility warning

HolaCarmensita commented:
Okey! Nu har jag lagt till en role=img till diven och även ändrat min h4 till en h3 för att behålla rätt semantik för bättre accessabilty. 

![Skärmavbild 2025-06-11 kl  08 23 26](https://github.com/user-attachments/assets/99d2261d-60cf-4f17-847b-3bbcaa9419a1)

HolaCarmensita committed:
feat: Add gallery functionality and new projects

- Added expandable gallery section for projects with multiple images
- Implemented conditional slide-up animation (disabled when gallery is open)
- Added The Pensieve project with 4 gallery images
- Added Coparenting App project with before/after design

HolaCarmensita committed:
feat: Update skills and toolbox with new tech

HolaCarmensita committed:
fix: space between number and rem

HolaCarmensita committed:
simplefied the fallback animation to vanilla css

HolaCarmensita committed:
alot of updates

HolaCarmensita committed:
ops i delited the more data, now added again

HolaCarmensita committed:
contact link styling

HolaCarmensita committed:
Update Netlify URL in projects.json

HolaCarmensita committed:
Update profile.json with new skills and tools

HolaCarmensita committed:
Revise profile description to emphasize AI interests

Updated the short description to reflect a focus on AI technologies and user-agent collaboration.

HolaCarmensita committed:
Update profile.json with new skills and tools

HolaCarmensita committed:
Revise short description in profile.json

Updated the short description to improve clarity and engagement.
